packages/cf-deploy-config-inquirer/src/prompts/app-router-prompts.ts

```diff
@@ -99,8 +99,8 @@ function getRouterTypePrompt(): CfAppRouterDeployConfigQuestions {
         default: () => RouterModuleType.Managed, // Should always be the preferred choice
         message: t('prompts.routerTypeMessage'),
         choices: [
-            { name: t('routerType.standaloneAppRouter'), value: RouterModuleType.Standard },
-            { name: t('routerType.managedAppRouter'), value: RouterModuleType.Managed }
+            { name: t('routerType.managedAppRouter'), value: RouterModuleType.Managed },
+            { name: t('routerType.standaloneAppRouter'), value: RouterModuleType.Standard }
```

Comment:
Might have CLI and/or E2E test implications?

Reply:
Ohhh yes! Thanks, I'll keep that in mind when I'm pulling in the change into TS.